media/static-bundles.json

```diff
@@ -1296,6 +1296,7 @@
         "js/base/core-datalayer.js",
         "js/base/core-datalayer-init.js",
         "js/base/search-params.js",
+        "js/base/uitour-lib.js",
```

Comment:
If we're adding this to the base bundle, then please add it to `common`, `common-protocol`, and `common-protocol-no-jquery`. It would then also need removing from any page-specific bundles to avoid including it twice.